views/partials/head.jade

```diff
@@ -0,0 +1,12 @@
+head
+	meta(charset='utf-8')
+	meta(http-equiv='X-UA-Compatible' content='IE=edge')
+	meta(name='viewport' content='width=device-width, initial-scale=1')
+	title= pageTitle
+	if styles && styles.length
+		each style in styles
+			link(as=style.as crossorigin=style.crossorigin href=style.href hreflang=style.hreflang media=style.media rel=style.rel sizes=style.sizes title=style.title type=style.type)
+	if scripts && scripts.length
+		each script in scripts
+			if script.inHead
+				script(async=script.async crossorigin=script.crossorigin defer=script.defer integrity=script.integrity src=script.src text=script.text type=script.type)
```

Comment:
I know this is an edge case, and I'm willing to let it go, but is there an easy way to allow the programmer to pass in other attributes and have them output with the tag? For example, how would a programmer create a `script` tag with the attribute `data-custom-attr="something"`? Can you look into if we can expose this someway in Jade?